lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
+// ====================
+
+// home - layout
+.container .discussion-body {
```

Comment:
Ah yeah, I can clean that up. Not everything in this existing file was nested within ".blank-slate" which is the existing "home" class. I can easily nest all other styles in this partial within .blank-slate however, which should make the scope clearer. (naming issue aside of "blank-slate"